File: Data Structures & Algorithms/remove-element/submission-2.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        i = 0
        k = 0
        while i < len(nums) - k:
            if nums[i] == val:
                k += 1
                for j in range(i + 1, len(nums)):
                    nums[j-1] = nums[j]
                continue
            i += 1
        return len(nums) - k

File: Data Structures & Algorithms/two-integer-sum/submission-2.go
func twoSum(nums []int, target int) []int {
    prevMap := make(map[int]int)
    for i, num := range nums {
        comp := target - num
        if j, ok := prevMap[comp]; ok {
            return []int{j, i}
        }
        prevMap[num] = i
    }
    return []int{}
}
